packages/catalog-process/test/utils.ts

```diff
@@ -276,3 +276,12 @@ export const readLastEventByStreamId = async (
     "SELECT * FROM catalog.events WHERE stream_id = $1 ORDER BY sequence_num DESC LIMIT 1",
     [eserviceId]
   );
+
+export const readLastTwoEventsByStreamId = async (
+  eServiceId: string,
+  postgresDB: IDatabase<unknown>
+): Promise<any> => // eslint-disable-line @typescript-eslint/no-explicit-any
+  await postgresDB.many(
+    "SELECT * FROM catalog.events WHERE stream_id = $1 ORDER BY sequence_num DESC LIMIT 2",
+    [eServiceId]
+  );
```

Comment:
> There is already the `readLastEventByStreamId` function, so we could either make that customizable by adding the limit as a parameter, or add a new function as shown here. The former solution would require to refactor all the usages of the previous function, to set the limit to 1.

How about refactor the function but having the `limit=1` as default?

Reply:
If we do as you suggest, these points would remain open:
- how will the function be called?
- if `limit` becomes a parameter, it means that the result will be an array in any cases, either of one or two items. But even for the cases where the array has one item, we will need to refactor all those usages to read the first item of an array, instead of the returned result.

Reply:
> If we do as you suggest, these points would remain open:
> 
>     * how will the function be called?
> 
>     * if `limit` becomes a parameter, it means that the result will be an array in any cases, either of one or two items. But even for the cases where the array has one item, we will need to refactor all those usages to read the first item of an array, instead of the returned result.

You're right, it could be complicated, maybe it would be better to refactor it with a parametric `limit`

Reply:
I added an intermediate solution to avoid refactors. Let me know what you think.